rfcs/0013-data-plane-fanout.md

```diff
@@ -0,0 +1,86 @@
+# Data Plane Fan-out 
+
+This RFC describes the current single layer broadcast and retransmit mechanisms as well as proposed changes to add a multi-layer retransmit via an Avalanche mechanism.
```

Comment:
Now that the RFC process is gone away, can you move this file into `book/src`?

Reply:
uff, yes missed that.